### DISCUSSION TITLE
release django framework integration guide

### DISCUSSION THREAD

**Discussion by Aliwahid17** [tailwindlabs/tailwindcss#8855] [Ideas]
I'm a Django developer and im struggling to use tailwind with my Django project.

Please explain how I can use tailwindcss with Django at the production level.

**Comment by theetherGit** [comment]
https://flowbite.com/docs/getting-started/django/

This is a pretty good explanation how to use django with tailwind

**Reply by Aliwahid17** [reply]
thanks for telling me I know about that but tailwindcss has to release a guide on their official website too.

**Reply by theetherGit** [reply]
Just add in the framework section and make a pull
